src/Model/GServer.php

```diff
@@ -1206,13 +1219,14 @@ private static function parseNodeinfo1(string $nodeinfo_url): array
 	 *
 	 * @see https://git.feneas.org/jaywink/nodeinfo2
 	 *
-	 * @param string $nodeinfo_url address of the nodeinfo path
+	 * @param string $nodeinfo_url     address of the nodeinfo path
+	 * @param int    $detection_method nodeinfo version
 	 *
 	 * @return array Server data
 	 *
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 */
-	private static function parseNodeinfo2(string $nodeinfo_url): array
+	private static function parseNodeinfo2(string $nodeinfo_url, int $detection_method): array
```

Comment:
What's the difference with `parseNodeinfo210`?

Reply:
What do you mean with that question?

Reply:
I expect `parseNodeinfo2` to process NodeInfo2 1.0, but there's also a `parseNodeinfo210` method, so I'm wondering what's the difference between both.

Reply:
That's why I renamed the functions.

Reply:
Oh, I see now, sorry for the noise.

Reply:
No problem.